src/Engine/PHP.php

```diff
@@ -160,9 +160,18 @@ protected function getCompilerFromPhpInfo($info)
         $compiler = '';
 
         foreach ($info as $s) {
-            if (false !== strpos($s, 'Compiler')) {
-                list(, $compiler) = explode('=>', $s);
-                break;
+            if (defined('PHP_WINDOWS_VERSION_MAJOR')) {
```

Comment:
Thanks!

The idea is correct however it may have to use the actual PHP used to install the extension, not the one used to called pickle. See https://github.com/FriendsOfPHP/pickle/blob/master/src/Engine/PHP.php#L71 for an example.

Reply:
Hmm, I think it does that. The check for `PHP_WINDOWS_VERSION_MAJOR` is just to detect Windows; since this is done in several other places already, I used it here as well. Otherwise I had used `PHP_OS_FAMILY === 'Windows'` (that constant is available as of PHP 7.2.0, so should be good).

Reply:
Oh Right, Not sure why I read VC in there :)

Reply:
np! Thanks for merging! :)